python/hsfs/core/feature_view_engine.py

```diff
@@ -1113,19 +1167,34 @@ def log_features(
                 for feature in fv.features
                 if feature.name not in td_predictions_names
             ]
-        df = engine.get_instance().get_feature_logging_df(
-            features_rows,
-            fg=fg,
-            td_features=td_features,
-            td_predictions=td_predictions,
-            td_col_name=FeatureViewEngine._LOG_TD_VERSION,
-            time_col_name=FeatureViewEngine._LOG_TIME,
-            model_col_name=FeatureViewEngine._HSML_MODEL,
-            predictions=predictions,
-            training_dataset_version=training_dataset_version,
-            hsml_model=hsml_model,
-        )
-        return fg.insert(df, write_options=default_write_options)
+
+        if http_data:
```

Comment:
Looking at the if-else condition, it http_data decides whether to return the features as list or DataFrame.
Could we rename the parameter for something a bit more intuitive? e.g., `as_list` or `as_dataframe`

Reply:
fixed